src/app.js

```diff
@@ -98,14 +104,16 @@ export const start = args => {
 
     doFetchBranches()
       .then(() => {
-        branchTable.clearItems();
```

Comment:
I thought it was a bit of duplication, since branchTable.clearItems() is called earlier on in the handler for C-r.

That being said, the removed clearItems() occurs in a promise, so it may be better to have it there in case the row data is populated before the callback is executed. I'll put it back!